lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -96,13 +96,20 @@
       - "The value that must be used here will be provided by a previous use
          of this module."
   dest:
-    description: The destination file for the certificate.
+    description:
+      - "The destination file for the certificate."
+      - "If C(fullchain_dest) is specified, C(dest) can be left away."
     required: true
     aliases: ['cert']
-  fullchain:
-    description: Include the full certificate chain in the destination file.
+  fullchain_dest:
+    description:
+      - "The destination file for the full chain (i.e. certificate followed
+         by chain of intermediate certificates)."
+      - "If C(dest) is specified, C(fullchain_dest) can be left away."
+    required: true
     default: false
```

Comment:
according code, there is no default, so can be left off.

Reply:
Oops, I just forgot to remove this. The type isn't `bool` anymore anyway, so `false` makes no sense.